src/server/worldserver/worldserver.conf.dist

```diff
@@ -3190,6 +3190,46 @@ AuctionHouseBot.forceIncludeItems = ""
 AuctionHouseBot.forceExcludeItems = ""
 
 #
+#    AuctionHouseBot.Class.RandomStackRatio.*
+#       Description: Used to determine how often an item of the class will be single or randomly-size stacked
```

Comment:
I think "an item" should be renamed to a "a stack" since some items cannot stack and therefore this description sounds misleading.